include/picongpu/plugins/PluginController.hpp

```diff
@@ -205,10 +205,7 @@ namespace picongpu
             plugins::transitionRadiation::TransitionRadiation<bmpl::_1>
 #if(ENABLE_OPENPMD == 1)
             ,
-            plugins::xrayScattering::XrayScattering<bmpl::_1>
-#endif
-#if(ENABLE_HDF5 == 1)
-            ,
+            plugins::xrayScattering::XrayScattering<bmpl::_1>,
             plugins::radiation::Radiation<bmpl::_1>,
```

Comment:
This should only be activated if HDF5_ENABLED is enabled.

Reply:
That is the `Radiation` plugin.